bool        request_control
bool        request_resume
bool        request_standby
bool        request_shutdown

float64     altitude
float64     heading
float64     speed
float64     angular_rate
float64     gps_time
bool        position_valid
bool        heading_valid
bool        speed_valid
bool        angular_rate_valid
float64     position_accuracy
